src/art.js

```diff
@@ -1712,15 +1712,25 @@
     "author_name": "Mayank Goel",
     "author_github_url": "https://github.com/yellowberard/"
   }
+
```

Comment:
The /src/art.js file should remain untouched. Please remove this so we can merged your pull request